activities/community-calls/preparing-community-call.md

```diff
@@ -1,6 +1,8 @@
 ---
 type: Guide
 explains: How to prepare a community call
+redirect_from:
+    - activities/codebase-stewardship/community-calls/preparing-community-call
 ---
 
 # Preparing a Standard for Public Code community call
```

Comment:
This header should probably be updated now when it is not specifically for the Standard community call anymore.